src/test/groovy/org/araneforseti/boundary/fields/EnumFieldTest.groovy

```diff
@@ -39,4 +42,9 @@ class EnumFieldTest {
     void field_uses_messageName() {
         test_both_messages(messageName)
     }
+
+    @Test
+    void field_uses_default_messageConfiguration() {
```

Comment:
Only the default message configuration is being tested here. It would be good to test the non-defaults too.